### ISSUE TITLE
Install cert-manager and nginx-ingress: .bat scripts links are not working

### ISSUE THREAD

**Issue by osamabinsaleem** [LM-Development/aks-sample#32]
In [this section ](https://github.com/LM-Development/aks-sample/blob/main/Samples/PublicSamples/RecordingBot/docs/deploy/aks.md#install-cert-manager-and-nginx-ingress) there are link to three files: 

- Modify [cluster-issuer.yaml](https://github.com/LM-Development/aks-sample/blob/main/Samples/PublicSamples/RecordingBot/deploy/cluster-issuer.yaml) by adding a contact [email](https://github.com/LM-Development/aks-sample/blob/main/Samples/PublicSamples/RecordingBot/deploy/cluster-issuer.yaml#L8).
- Run the [cert-manager.bat](https://github.com/LM-Development/aks-sample/blob/main/Samples/PublicSamples/RecordingBot/deploy/cert-manager.bat) script.
- Run the [ingress-nginx.bat](https://github.com/LM-Development/aks-sample/blob/main/Samples/PublicSamples/RecordingBot/deploy/ingress-nginx.bat) script.

All of these links are not working and also If I search for these files they are present in the deprecated folder as show in the image below:
<img width="317" alt="Screenshot 2024-03-06 at 2 03 05 PM" src="https://github.com/LM-Development/aks-sample/assets/37154675/8db324f6-f7a0-4dab-b85b-4ffbf4b49f76">

Are these new files ported to the the new folder with different names?

**Comment by InDieTasten** [comment]
For the cert bot, take a look here: https://github.com/LM-Development/aks-sample/tree/main/Samples/PublicSamples/RecordingBot/deploy/cert-manager

The ingress has been moved into the dependencies of the application chart here: https://github.com/LM-Development/aks-sample/blob/main/Samples/PublicSamples/RecordingBot/deploy/teams-recording-bot/Chart.yaml

So the installation step can be skipped and will be applied automatically, once you get to the step of deploying the helm chart itself.

Docs will need to be changed in that regard. We will probably also undo the packaging of nginx-ingress as dependency and have that separate again in the future. But for now, that approach should work.

**Comment by InDieTasten** [comment]
@osamabinsaleem Yes, just the cert bot. The ingress will be deployed automatically has part of the application helm chart.
